### DISCUSSION TITLE
Is there a way to modify minecraft ore generation without disabling it?

### DISCUSSION THREAD

**Discussion by ShadowDragon1848** [EwyBoy/OreTweaker#94] [Help Wanted!] [answered]
Hey, I have a question. I wanted to make a few small tweaks to MC ore generation and found this mod. But I do find it a bit overwhelming to disable all the ores I want to tweak and then to write all that for every ore only to make a few small changes. Is it possible to modify the "normal" ore generation? meaning to override it? For example MC spawns diamonds very deep down and I want make it appear direct on the surface, could I just write something like: 
      "minY": 50,
      "maxY": 70,
?

**Comment by EwyBoy** [comment, marked as answer]
You can simply tweak the ore generation out of the box for vanilla ores. It will also work for some modded ores but not all. It all depends on how the author of the mod implemented their ore generation.

You can give it at try and it might work, or it might not.

It's not a ton of work to go into the mods config file either and disable the ores and copy paste some json files and change a few of the values.

**Reply by ShadowDragon1848** [reply]
How do I modify the MC generation? I´m sorry, I´m not that familiar with coding.

**Reply by EwyBoy** [reply]
Simply open the files, and change the values. Check the wiki for more info.

**Reply by ShadowDragon1848** [reply]
Okay
